index.html

```diff
@@ -259,11 +259,12 @@
           </p>
           <ul>
             <li>
-              <code>http://fedora.info/definitions/repository#EmbedResources</code>:
-              Requires a server to include representations of any contained resources in the response.
+              <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>:
```

Comment:
Thanks, @azaroth42 . That seems very much in line with @acoburn's suggestion.
:+1: